test/e2e/framework/util.go

```diff
@@ -3450,6 +3450,11 @@ func GetSigner(provider string) (ssh.Signer, error) {
 		if len(keyfile) == 0 {
 			keyfile = "id_rsa"
 		}
+	case "skeleton":
+		keyfile = os.Getenv("KUBE_SSH_KEY")
```

Comment:
is `KUBE_SSH_KEY` an absolute path? this function is kinda weird and probably needs to be refactored, since it prepends `keydir` to this path before reading it - compare the gce vs aws cases, for example.

Reply:
KUBE_SSH_KEY will have just a name of key file, which will be combined later with the location of .ssh folder